planner/cascades/testdata/integration_suite_in.json

```diff
@@ -28,7 +28,9 @@
       "select b, sum(a) from t group by b order by b",
       "select b, avg(a) from t group by b having sum(a) > 1 order by b",
       "select max(a+b) from t",
-      "select sum(a) from t group by a, a+b order by a"
+      "select sum(a) from t group by a, a+b order by a",
+      "select max(a) from t",
+      "select max(a), b from t"
```

Comment:
Add test case `select min(a) from t` and `select b, min(a) from t`. 

To test whether the cost model can choose the best plan, we also need to test `select min(b) from t`.